addons/safemode/functions/fnc_playChangeFiremodeSound.sqf

```diff
@@ -16,12 +32,13 @@ if ({(toLower (_sound select 0) find _x == (count toArray (_sound select 0) - co
 };
 
 // add default volume, pitch and distance
-if (count _sound < 2) then {_sound pushBack 1};
-if (count _sound < 3) then {_sound pushBack 1};
-if (count _sound < 4) then {_sound pushBack 0};
+_soundCount = count _sound;
+if (_soundCount < 2) then {_sound pushBack 1};
+if (_soundCount < 3) then {_sound pushBack 1};
+if (_soundCount < 4) then {_sound pushBack 0};
```

Comment:
This is wrong. `count _sound` increases after `pushBack`, but `_soundCount` does not.